schedule.Rmd

```diff
@@ -41,27 +43,27 @@ Every class session will start and end with a writing prompt that will be used f
 </div>
 
-##### **Week 4**
+##### **4 (`r config::get("term_dates_1")$wk4`)**
 
 **Project Planning and Forecasting**
 
 <div class = "table_style">
 |**Summary**| **Content** |**Homework <br> (please do before next week)**|**Resources**|
 |--------------|------------|-------------|--------------|
-|This week we will discuss more specifically about how students will **interact** with partners, as well as examples of what has and has not worked well in past similar projects. Students will meet with their **project group** for the first time and start assigning **student roles**.|* [Project planning form](https://docs.google.com/document/d/1b0XHuKsys_ACCtA9nI_M9F1CR2oWo2RLR-A1C6cvz0U/edit?usp=sharing) <br> * [Project organization and infrastructure suggestions](https://docs.google.com/document/d/1RveTbckBTomgx2JiiCbm01SE88OvODK7JCQxo1gbLbo/edit?usp=sharing)  |  * [Git & GitHub Assignment](https://jhudatascience.org/Baltimore_Community_Course/Assignments/Using_git_Assignment.html) |  * [SOURCE Dos and don'ts](https://source.jhu.edu/publications-and-resources/DosAndDontsOfPartnerships.pdf) <br> * [Project Student Roles](https://jhudatascience.org/Baltimore_Community_Course/projects.html) <br> * [Project Reminders](https://jhudatascience.org/Baltimore_Community_Course/projects.html)|
+|This week we will discuss more specifically about how students will **interact** with partners, as well as examples of what has and has not worked well in past similar projects. Students will meet with their **project group** for the first time and start assigning **student roles**.|* Minimum Viable Products (MVPs) <br> * [Project planning form](https://docs.google.com/document/d/1b0XHuKsys_ACCtA9nI_M9F1CR2oWo2RLR-A1C6cvz0U/edit?usp=sharing) <br> * [Project organization and infrastructure suggestions](https://docs.google.com/document/d/1RveTbckBTomgx2JiiCbm01SE88OvODK7JCQxo1gbLbo/edit?usp=sharing)  |  * [Git & GitHub Assignment](https://jhudatascience.org/Baltimore_Community_Course/Assignments/Using_git_Assignment.html) |  * [SOURCE Dos and don'ts](https://source.jhu.edu/publications-and-resources/DosAndDontsOfPartnerships.pdf) <br> * [Project Student Roles](https://jhudatascience.org/Baltimore_Community_Course/projects.html) <br> * [Project Reminders](https://jhudatascience.org/Baltimore_Community_Course/projects.html)|
```

Comment:
is there an MVP link we could use? or maybe that will be our own slides?

Reply:
Yes, will add later. Just adding as a placehodler for now.